modules/nw-sriov-installing-operator.adoc

```diff
@@ -64,45 +59,36 @@ metadata:
 spec:
   targetNamespaces:
   - openshift-sriov-network-operator
+EOF
 ----
 
-.. To create the OperatorGroup CR, enter the following command:
-+
-----
-$ oc create -f sriov-operatorgroup.yaml
-----
+. Subscribe to the SR-IOV Network Operator.
 
-.. Run the following command to get the `channel` value required for the next
+.. Run the following command to get the {product-title} major and minor version. It is required for the `channel` value in the next
 step.
 +
+[source,terminal]
 ----
-$ oc get packagemanifest sriov-network-operator -n openshift-marketplace \
-  -o jsonpath='{.status.defaultChannel}'
+$ OC_VERSION=$(oc version -o yaml | grep openshiftVersion | \
```

Comment:
@jboxman  could you please help me understand why the `packagemanifest` cmd is not used?

Reply:
@zshi-redhat, I've been trying to understand that myself, for months.

So previously, `defaultChannel` is used. Somehow, that is defined either by the Operator developer or possibly OLM; but in either case, it is wrong for some versions of OCP. For example: https://bugzilla.redhat.com/show_bug.cgi?id=1849781

So I keep trying to find some way to provide the correct information for every release of OCP that someone might install the Operator.

If this isn't valid, then I'll try using `{product-version}` for the value at docs build time; And if that doesn't work, I can go back to listing every single `channel` and asking the user to type the correct channel value when creating the CR;

But I'd rather that someone can simply copy and paste the correct CR, and run it, and move on. Why not automate as much as possible, the procedure for a user?

Reply:
/cc @zhaozhanqi 

It looks like defaultChannel is always set to latest channel (that's available for subscription), so when user tries to deploy Operator in a 4.4 openshift environment, they get 4.5 from defaultChannel.

I think using product-version might be the appropriate way to get the right Operator version for a deployed cluster.

One possible issue of using `oc version -o yaml` might be if user doesn't update the oc binary after upgrading the cluster, then it will show a version number that's lower than the deployed cluster. (this may not matter much or happen frequently as it is likely  due to a mis-operation during the upgrade)

Reply:
@zshi-redhat 
`oc version -o yaml | grep openshiftVersion` will show the server version, not client oc version.
I checked this `oc version -o yaml | grep openshiftVersion | grep -o '[0-9]*[.][0-9]*' | head -1`, it LGTM

Reply:
> @zshi-redhat
> `oc version -o yaml | grep openshiftVersion` will show the server version, not client oc version.
> I checked this `oc version -o yaml | grep openshiftVersion | grep -o '[0-9]*[.][0-9]*' | head -1`, it LGTM

Thanks for checking @zhaozhanqi , if it returns the server version, then we should have no problem using `oc version.`

Reply:
It does seem to be the server version itself:

```
clientVersion:
  buildDate: "2020-04-26T20:18:56Z"
  compiler: gc
  gitCommit: 2576e482bf003e34e67ba3d69edcf5d411cfd6f3
  gitTreeState: dirty
  gitVersion: openshift-clients-4.4.0-202004250654
  goVersion: go1.13.4
  major: ""
  minor: ""
  platform: darwin/amd64
openshiftVersion: 4.4.3
releaseClientVersion: 4.4.3
serverVersion:
  buildDate: "2020-04-26T20:16:35Z"
  compiler: gc
  gitCommit: b9b84e0
  gitTreeState: clean
  gitVersion: v1.17.1
  goVersion: go1.13.4
  major: "1"
  minor: 17+
  platform: linux/amd64
```

Reply:
@zshi-redhat, confirmed with a newer client. The server version is correct:

```
clientVersion:
  buildDate: "2020-07-17T21:59:28Z"
  compiler: gc
  gitCommit: b66f2d3a6893be729f1b8660309a59c6e0b69196
  gitTreeState: clean
  gitVersion: openshift-clients-4.5.0-202006231303.p0-4-gb66f2d3a6
  goVersion: go1.13.4
  major: ""
  minor: ""
  platform: darwin/amd64
openshiftVersion: 4.4.3
releaseClientVersion: 4.5.3
serverVersion:
  buildDate: "2020-04-26T20:16:35Z"
  compiler: gc
  gitCommit: b9b84e0
  gitTreeState: clean
  gitVersion: v1.17.1
  goVersion: go1.13.4
  major: "1"
  minor: 17+
  platform: linux/amd64
```